content/guides/translation-connection/step3-interact-with-julia.md

```diff
@@ -0,0 +1,241 @@
+
+`Translation status`  
+**Attribute type:** Simple select  
+**Attribute code:** translationStatus  
+**Attribute group:** Translations  
+**Value per locale:** Enabled (in order to have a status for each locale)    
+**Read only:** Enabled (Can only be modified via API)   
+**Usable in grid:** Enabled    
+
+This attribute contains these options:    
+|      Code      |     Label      |
+| :------------- | :------------- |
+| INPROGRESS     | IN PROGRESS    |
+| TRANSLATED     | TRANSLATED     |
+| CANCELLED      | CANCELLED      |
+
+As this last attribute has a `Read only` property, it can't be modified by Julia. Only your connector can modify this status through API and set the translation project status for each desired locale.
+
+:::info
+This list of attributes is not exhaustive and may need to be adapted depending on your online Translation solution and the information required to perform a translation project.
+:::
+
+Once the translation attribute group and attributes have been created, Julia needs to assign the `Translations` attribute group to all existing `families`.
+
+:::tips
+Julia can do this by using this [bulk action](https://help.akeneo.com/pim/serenity/articles/product-mass-actions.html) process:
+1. Go to `Settings > Families`
+2. Select all families
+3. Click on `BULK ACTIONS button` at bottom
+4. Select `Set attributes requirements` and then click on `NEXT`
+5. Click on `ADD BY GROUPS` drop down option and `ADD` `Translations` attribute group
+6. Click on `CONFIRM`
+:::
+
+>And that's it! Congratulations, everything is now ready to receive Julia's PIM products she wants to translate!
+
+## Julia's translation process
+
+As you have understood, Julia will use the previous PIM attributes to define which of her products are to be translated and how they will be processed.
+
+Now let's see how Julia will work with these new PIM translation features.
+
+### Select the attributes to be translated
+
+First, Julia needs to select product she wants to translate:
+1. Go to `Products` menu
+2. Select the "source" `locale`
+3. Select products (and/or product models) you want to translate
+
+:::tips
+For this product selection, Julia can help herself by using her PIM grid [filtering](https://help.akeneo.com/pim/serenity/articles/products-grid.html#use-filters) system or by creating a [specific view](https://help.akeneo.com/pim/serenity/articles/manage-your-views.html).
+:::
+
+### Perform a "Bulk action"
+
+Then, she needs to perform a [Bulk action](https://help.akeneo.com/pim/serenity/articles/product-mass-actions.html) on these products.
+
+For this action, Julia have to:
+1. Click on `BULK ACTIONS` button at bottom
+2. Select `Edit attribute`s values and click on `NEXT`.
+3. From `SELECT ATTRIBUTES` drop down, search and select these 5 translation attributes and click on `ADD`:
+  * Translation submission name
+  * Translation submitter
+  * Translation queued
+  * Translation locales
+  * Translation due date
```

Comment:
```suggestion
    * Translation submission name
    * Translation submitter
    * Translation queued
    * Translation locales
    * Translation due date
```
You need to indent twice to put a list in another list (here, the numbered list), otherwise the list is not included inside the numbered list and thus, the next numbered item will reset its counter to 1.